.github/workflows/build.yml

```diff
@@ -0,0 +1,50 @@
+on:
+  push:
+    branches:
```

Comment:
we probably need to run this workflow for every push to `master` and `branch-`

so you can add something like
```
branches:
  - master
  - branch-*.*
 ```

Reply:
Added branch list.